src/functions/files.ts

```diff
@@ -280,6 +314,128 @@ export const reUploadImageToS3AndRemove = async (startAt: number = 0) => {
   }
 }
 
+/**
+ * flow this document https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#listObjects-property
```

Comment:
use `@see` directive

```suggestion
 * @see https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#listObjects-property
```